docs/panel/js/pages/commands/default.js

```diff
@@ -176,7 +176,8 @@ $(function() {
                                     'If checked, the command cannot be used in chat.'))
                                 // Callback function to be called once we hit the save button on the modal.
                         })), function() {
-                            let commandPermission = $('#command-permission'),
+                            let commandName = $('#command-name'),
+                                commandPermission = $('#command-permission'),
```

Comment:
Please do not edit files in the `docs/panel/` folder. This folder is updated by automation upon merge